exampleCourse/questions/demo/surveyQuestion/question.html

```diff
@@ -0,0 +1,47 @@
+<pl-question-panel>
+
+    <p>Please answer the following demographic survey questions. </p>
+    <p>As this is a survey, there are no correct or incorrect responses.</p>
+
+    <hr>
+
+    <h6>Race/Ethnicity: Check all that apply</h6>
+
+    <p><label><input type="checkbox" name="race" value="white" />White or European American</label></p>
+    <p><label><input type="checkbox" name="race" value="black" />Black or African American</label></p>
+    <p><label><input type="checkbox" name="race" value="hispanic" />Hispanic, Latina/o, or
+            Spanish</label></p>
+    <p><label><input type="checkbox" name="race" value="asian" />Asian American</label></p>
+    <p><label><input type="checkbox" name="race" value="pacific" />Native Hawaiian or Other Pacific
+            Islander</label></p>
+    <p><label><input type="checkbox" name="race" value="native" />Native American or Alaskan
+            Native</label></p>
+    <p>Other (please specify): <input type="text" name="otherrace" /></p>
+
+    <hr>
+
+    <h6>Gender: Select one</h6>
+
+    <p><label><input type="radio" name="gender" value="male" />Male</label></p>
+    <p><label><input type="radio" name="gender" value="female" />Female</label></p>
+    <p><label><input type="radio" name="gender" value="nonbin" />Non-Binary</label></p>
```

Comment:
May add a "prefer not to answer"?

Reply:
Done here https://github.com/PrairieLearn/PrairieLearn/pull/1867/commits/f41c4e6e0e49da34753af17132c3fb0484b9df2c